skt/executable.py

```diff
@@ -341,15 +341,20 @@ def cmd_build(args):
     # Gather additional details about the build and save them to the state
     # file.
     kernel_arch = builder.build_arch
-    cross_compiler_prefix = builder.cross_compiler_prefix
```

Comment:
Mmm it wouldn't be enough something like this?
```python
     cross_compiler_prefix = builder.cross_compiler_prefix or ''
```

Reply:
Strictly speaking, it won't become `None` after reading, so won't be equivalent, and can result in confusion either in computers or in humans.